### ISSUE TITLE
Email notifications

### ISSUE THREAD

**Issue by ocdtrekkie** [synchrone/sandstorm-radicale#25]
It'd be super awesome if I could get email notifications for calendar events. This is probably the top way I interact with my calendar.

**Comment by synchrone** [comment]
This is actually a feautre I would love to have myself.
Unfortunately, Sandstorm suspends the grain when it is not in use (i.e not open in a browser, and no API interaction).

What would make this possible is some sort of a cron facility, where apps could register to be woken up on a rough schedule. Then we could handle sending the emails from the backend.

**Comment by ocdtrekkie** [comment]
Right. I knew that. :/ I am going to explicitly tag @paulproteus so he is aware of this use case for such a feature.

**Comment by ocdtrekkie** [comment]
As a note, a cron API lives in an unmerged PR, so with any luck (and some hope), someday soon this will be possible. https://github.com/sandstorm-io/sandstorm/pull/2922
